mosaic-site/src/layouts/DetailTechnical/DetailTechnical.module.css

```diff
@@ -16,6 +16,36 @@
   --border-width-medium: 3px;
 }
 
+.primarySidebar :global(.ps-menu-button) {
+  --color-dark-neutral-background-emphasis: var(--salt-palette-navigate-primary-background-hover);
+  --color-dark-navigable-selectableLink-unselectedLabel: var(--salt-palette-neutral-secondary-foreground);
```

Comment:
```suggestion
  --color-dark-navigable-selectableLink-unselectedLabel: var(--salt-text-secondary-foreground);
```

Reply:
not sure about these ones, if mosaic color tokens are palette, it might be ok to keep as palette tokens, but in general we should only use characteristic tokens